internal/api/utils.go

```diff
@@ -324,19 +310,20 @@ func transformComponentUnits(recommendationJSON map[string]interface{}) map[stri
 
 						sectionObject, ok := recommendationSection[section].(map[string]interface{})
 						if ok {
-							memory, ok := sectionObject["memory"].(map[string]interface{})
+							memInMi, ok := sectionObject["memory"].(map[string]interface{})
 							if ok {
-								err := convertMemory(memory)
-								if err != nil {
-									fmt.Printf("error converting memory in %s: %v\n", period, err)
+								err := truncateMemory(memInMi)
+								if !err {
+									fmt.Printf("error truncating the memory in %s: %v\n", period, err)
 									continue
 								}
 							}
+
 							cpu, ok := sectionObject["cpu"].(map[string]interface{})
 							if ok {
-								err := convertCPU(cpu)
-								if err != nil {
-									fmt.Printf("error converting cpu in %s: %v\n", period, err)
+								err := truncateCPU(cpu)
+								if !err {
+									fmt.Printf("error truncating cpu in %s: %v\n", period, err)
```

Comment:
If possible, please help with convert this `fmt.Printf` statements to `log.Error` 🙂